### PR TITLE
Fix up some pedant spec descriptions that lagged behind API developments

### PR DESCRIPTION
**Pull request by randomcamel**

### CODE REVIEW

**Comment by stevendanna** [comment]
:+1: This looks reasonable to me.

File: oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
   include Pedant::RSpec::CookbookUtil
 
-  context "DELETE /cookbooks/<name>/<version>" do
+  context "DELETE /cookbook_artifacts/<name>/<version>" do
```
Comment (stevendanna) [inline oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb]: Minor quibble but below we call "version" a "cookbook_identifier".  Perhaps that is the better name?

Comment (randomcamel) [inline oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb]: It is better, and I'll fix it. Due to some conceptual and code copy-paste, the code and tests often play fast and loose with terminology.

Comment (stevendanna) [inline oc-chef-pedant/spec/api/cookbook_artifacts/delete_spec.rb]: @randomcamel I'm fine leaving that call to you or @danielsdeleo who know this code better than I most likely.

**Pull request merged**